packages/website/src/bin/integration.test.js

```diff
@@ -17,7 +17,7 @@ const customBabelConfig = path.join(
   'custom-babel-project/docs.config',
 );
 
-describe('Website generation integration tests', () => {
+describe.skip('Website generation integration tests', () => {
```

Comment:
Do you plan on coming back to fix this? If not I recommend deleting the tests altogether.

Reply:
yes, will be deleting as we delete the old page generation logics completely